### PR TITLE
Pacman

### PR DESCRIPTION
**Pull request by RaaVanansri**
I added a npm package called "react-pacman" and a SASS/SCSS loader in webpack.config.js

Last time i made a mistake in webpack.config.js file,so i closed that request :)

### CODE REVIEW

**Comment by RaaVanansri** [comment]
Closes the issue #236

File: apps/pacman/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-community/pacman",
+  name: "pacman",
+  description: "pacman game intergration with felvin",
+  queryToData,
+  Component,
+  // screenshotPath: "./files/screenshot.png",
```
Comment (hargup) [inline apps/pacman/src/index.ts:10]: @RaaVanansri can you add the screenshot of the app here, thanks :)

File: apps/pacman/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  // exampleSearchQueries: [],
```
Comment (hargup) [inline apps/pacman/src/index.ts:11]: Also the example queries.

**Review by hargup — APPROVED** [review]
LGTM

**Pull request merged**